util/util.go

```diff
@@ -230,3 +230,13 @@ func GetTransitGatewayLocationAndRouting(powerVSZone *string, vpcRegion *string)
 	// since VPC region is not set and used PowerVS region to calculate the transit gateway location, hence returning local routing as default.
 	return &location, ptr.To(false), nil
 }
+
+// VPCZonesForVPCRegion returns the VPC zones associated with the VPC region.
+func VPCZonesForVPCRegion(region string) ([]string, error) {
+	for _, regionalComponentsData := range Regions {
```

Comment:
Can we rename to a simpler name as `RegionDetails` or `RegionData`?

Reply:
Yes, can be simplified further.
I have switched to RegionDetails.

Thanks!